src/lib/viewers/doc/DocFindBar.js

```diff
@@ -137,12 +138,19 @@ class DocFindBar extends EventEmitter {
      * @return {void}
      */
     dispatchFindEvent(type, findPrev) {
-        this.findController.executeCommand(type, {
-            query: this.findFieldEl.value,
-            phraseSearch: true, // true by default
-            highlightAll: true, // true by default
-            findPrevious: findPrev,
-        });
+        return Browser.isIE()
+            ? this.findController.executeCommand(type, {
```

Comment:
Do we need to duplicate the options object being passed or can it be moved to a variable?